docker-compose.yml

```diff
@@ -43,6 +43,13 @@ services:
       - ./server:/apps/warehouse/server
       - ./package.json:/apps/warehouse/package.json
       - ./npm-shrinkwrap.json:/apps/warehouse/npm-shrinkwrap.json
+    depends_on:
+      - db
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://web:3000/api/StoreModels"]
```

Comment:
> you can provide --start-period=DURATION for healthchecks to start.
@harshadyeola - i like it but it is not supported for v2.1 of docker-compose.yml file, i tried two different variations:
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.web.healthcheck value Additional properties are not allowed
('start-period' was unexpected)
...
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.web.healthcheck value Additional properties are not allowed
('start_period' was unexpected)
```

How about I increase the 15 second interval to be 45 seconds so that it will more or less give us the same results as `--start-period=60s`